documentation/src/main/asciidoc/topics/config_examples/remote-store.json

```diff
@@ -0,0 +1,51 @@
+{
+  "remote-store": {
```

Comment:
Serializing the entire persistence configuration seems overkill. Can we not just serialize a RemoteStoreConfiguration instead? Also this means anyone could add a different store impl this way, which seems a bit out of scope for the `connectSource` method currently.

Reply:
As I wrote above, at REST API level, it probably makes sense to allow 1 and only 1 ```remote-store``` while at PM level it would allow only 1 ```StoreConfiguration```